eox_tenant/models.py

```diff
@@ -65,6 +66,59 @@ def get_microsite_for_domain(cls, domain):
         return microsites[0] if microsites else None
 
 
+class TenantConfig(models.Model):
+    """
+    Model to persist edxapp configurations.
+    """
+    key = models.CharField(max_length=63, db_index=True)
+    values = JSONField(null=False, blank=True, load_kwargs={'object_pairs_hook': collections.OrderedDict})
```

Comment:
why do we keep the values field. This should be replaced by lms_configs and completely remove values since it causes a lot of confusion with the old microsites.